src/lib.rs

```diff
@@ -0,0 +1,132 @@
+// MinSQL
+// Copyright (C) 2019  MinIO
```

Comment:
We should follow the GNU AGPL style license headers 

```
/* 
 * This file is part of the MinSQL
 * Copyright (c) 2019 MinIO, Inc.
```

Reply:
This should be changed across files

Reply:
I was using this style https://github.com/licenses/license-templates/blob/master/templates/agpl3-header.txt

but I can update to your suggestion